### ISSUE TITLE
Can the 'choose the storage location' / login popup be supressed til it's required?

### ISSUE THREAD

**Issue by JPL1** [theWebalyst/solid-filemanager#2]
Reloading a site or navigating to a new site by pasting the URL into the query string requires okaying it in the storage location popup again ('open directory'). Maybe this won't happen once it is possible to login, but it would be good if this popup only appeared when required.

**Comment by Otto-AA** [comment]
I will add a check for the login status before the popup appears. Should be done within the next days

> Maybe this won't happen once it is possible to login

I am not sure if I understand you here. At least for the Solid version, login is possible already. Maybe that's different for the Safe version though

**Comment by happybeing** [comment]
Yes there's a bug in SAFE Browser which currently prevents login to SAFE Network - so access is currently read only.

**Comment by Otto-AA** [comment]
> I will add a check for the login status before the popup appears. Should be done within the next days

This change will not come before the whole situation with the NSS server and the release of solid-file-client v1.0.0 is finished. I want to get everything working again properly before doing any other changes to solid-filemanager. So this likely won't get resolved within the next weeks, maybe (hopefully) in October. I've also added a note to the solid-filemanager repo about the current status of this project. I'm sorry for any inconvenience which may be caused by this to you and others using this app.

**Comment by happybeing** [comment]
This is no problem at all @Otto-AA but thanks for keeping us up to date. 

FYI I'll be publicising a blog post about using Solid Filemanager on SAFE Network later today (written by a member of the SAFE community). Will announce it on Solid platforms.
